README.md

```diff
@@ -829,14 +829,31 @@ or send an email to [info@4r7.ir](mailto:info@4r7.ir "info@4r7.ir").
 
 * Logo designed by [Arta Khanalizadeh](https://www.linkedin.com/in/artakhanalizadeh)	
 
-## Donate to our project
+## Show your support
 								
-<h3>Bitcoin :</h3>					
-
-```12Xm1qL4MXYWiY9sRMoa3VpfTfw6su3vNq```			
-
-
-
-<h3>Payping (For Iranian citizens)</h3>
-
-<a href="http://www.payping.net/sepandhaghighi" target="__blank"><img src="http://www.qpage.ir/images/payping.png" height=100px width=100px></a>	
+<h3>Star this repo</h3>					
+
+Give a ⭐️ if this project helped you!
+
+<h3>Donate to our project</h3>	
+
+<h4>Bitcoin</h4>
+`1KtNLEEeUbTEK9PdN6Ya3ZAKXaqoKUuxCy`
+<h4>Ethereum</h4>
+`0xcD4Db18B6664A9662123D4307B074aE968535388`
+<h4>Litecoin</h4>
+`Ldnz5gMcEeV8BAdsyf8FstWDC6uyYR6pgZ`
+<h4>Doge</h4>
+`DDUnKpFQbBqLpFVZ9DfuVysBdr249HxVDh`
+<h4>Tron</h4>
+`TCZxzPZLcJHr2qR3uPUB1tXB6L3FDSSAx7`
+<h4>Ripple</h4>
+`rN7ZuRG7HDGHR5nof8nu5LrsbmSB61V1qq`
+<h4>Binance Coin</h4>
+`bnb1zglwcf0ac3d0s2f6ck5kgwvcru4tlctt4p5qef`
+<h4>Tether</h4>
+`0xcD4Db18B6664A9662123D4307B074aE968535388`
+<h4>Dash</h4>
+`Xd3Yn2qZJ7VE8nbKw2fS98aLxR5M6WUU3s`
+<h4>Stellar</h4>
+`GALPOLPISRHIYHLQER2TLJRGUSZH52RYDK6C3HIU4PSMNAV65Q36EGNL`
```

Comment:
Nice job.
Can't we make some link kind of link which direct who wants to donate to a special website which he/she can donate us much easier than copy-paste the wallet?

Reply:
We only accept cryptocurrency, so there is no problem ;-)